include/convex_bodies/ellipsoid.h

```diff
@@ -28,12 +28,13 @@ typedef Point PointType;
     typedef Eigen::Matrix<NT, Eigen::Dynamic, 1> VT;
     typedef Eigen::Matrix<NT, Eigen::Dynamic, Eigen::Dynamic> MT;
 
+private:
     // representation is (x - c)' A (x - c) <= 1, center is assumed to be origin for now
     MT A;
     Point c;
```

Comment:
In your implementation, the center is always the origin. Do you really need this variable?

Reply:
removed the variable 👍🏽